sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1979,6 +1979,15 @@ object SQLConf {
       .doc("When true, the ArrayExists will follow the three-valued boolean logic.")
       .booleanConf
       .createWithDefault(true)
+
+  val ADDITIONAL_REMOTE_REPOSITORIES =
+    buildConf("spark.sql.additionalRemoteRepositories")
+      .doc("A comma-delimited string config of the optional additional remote maven mirror " +
```

Comment:
No big deal, but if you make another round of edits:
maven -> Maven
, this -> . This
maven central -> Maven Central

Reply:
Sure, thanks for the comment, done in 6e6b87c.